services/core/FileWatchPublisher/IDENTITY

```diff
@@ -0,0 +1 @@
+platform.filewatchpublisher
```

Comment:
add {n} to the end of this...That will allow us to install more than one of these agents...